Watchman.Discord/Framework/Workflow.cs

```diff
@@ -92,12 +92,24 @@ private Task RunControllers(SocketMessage message)
 
                     if (message.Content.ToLowerInvariant().StartsWith(command.Command))
                     {
+                        bool isMethodAdminOnly = method.CustomAttributes.Any(a => a.AttributeType.FullName == typeof(AdminCommand).FullName);
+
+                        if (isMethodAdminOnly && !HasAdminPermissions(message.Author))
+                        {
+                            break;
```

Comment:
```suggestion
                            continue;
```
to że akurat ta metoda tego wymaga, nie znaczy że ma nie sprawdzać reszty

Reply:
a nie
dobrze było
break; ma być